### PR TITLE
Document and Implement Tensorflow Lite interpreter

### PR DESCRIPTION
**Pull request by seanohair22**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/retail-use-cases/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable

- In root level directory, run `make update-submodules download-sample-media`
- Do `cd use-cases/demos` to navigate to correct dir
- Change the `USE_TFLITE=0` to 1 to use Tensorflow Lite interpreter within the `use-cases/demos/tensorflow-keras-env.txt` file
- Run `make run_tensorflow_keras_classification`
- You should see an inference window pop-up on screen with a coca-cola video and either the title `TensorflowLite-Keras Classification` or `Tensorflow-Keras Classification` depending on whether you selected to use TFLite or TF

### CODE REVIEW

File: use-cases/demos/classification/tensorflow-keras/entrypoint.sh [hunk 1]
```diff
@@ -9,7 +9,12 @@
 cid=$(date +%Y%m%d%H%M%S%N)
 echo "cid: $cid"
 
-PROFILE_NAME="demo_tensorflow_keras_classification"
-
-python3 classification/tensorflow-keras/tensorflow_keras_classification.py \
--m "$MQTT_HOSTNAME" -p "$MQTT_PORT" -t "$MQTT_TOPIC" -i "$INPUT_SRC" | tee >/tmp/results/r"$cid"_"$PROFILE_NAME".jsonl >(stdbuf -oL sed -n -e 's/^.*FPS: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline"$cid"_"$PROFILE_NAME".log)
+if [ $USE_TFLITE -eq 1 ]; then
+    PROFILE_NAME="demo_tensorflow_lite_keras_classification"
+    python3 classification/tensorflow-keras/liteRT_keras_classification.py \
```
Comment (NeethuES-intel) [inline use-cases/demos/classification/tensorflow-keras/entrypoint.sh:14]: Sean, follow the same Detection_Script strategy in my PR to avoid the code duplication - https://github.com/intel-retail/retail-use-cases/blob/dc61063d95d0908b7d515e176fcfbdac8f080c62/use-cases/demos/object_detection/pytorch-yolo/entrypoint.sh#L26

**Review by NeethuES-intel — CHANGES_REQUESTED** [review]

**Review by jim-wang-intel — APPROVED** [review]
tested and worked for me- just that on my box, the first time took about 30-50 seconds for the pop-up window to show up.

File: use-cases/demos/classification/tensorflow-keras/entrypoint.sh [hunk 2]
```diff
@@ -9,7 +9,12 @@
...
-PROFILE_NAME="demo_tensorflow_keras_classification"
-
-python3 classification/tensorflow-keras/tensorflow_keras_classification.py \
--m "$MQTT_HOSTNAME" -p "$MQTT_PORT" -t "$MQTT_TOPIC" -i "$INPUT_SRC" | tee >/tmp/results/r"$cid"_"$PROFILE_NAME".jsonl >(stdbuf -oL sed -n -e 's/^.*FPS: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline"$cid"_"$PROFILE_NAME".log)
+if [ $USE_TFLITE -eq 1 ]; then
+    PROFILE_NAME="demo_tensorflow_lite_keras_classification"
+    DETECTION_SCRIPT="classification/tensorflow-keras/liteRT_keras_classification.py"
+else
+    PROFILE_NAME="demo_tensorflow_keras_classification"
+    DETECTION_SCRIPT="classification/tensorflow-keras/tensorflow_keras_classification.py"
```
Comment (jim-wang-intel) [inline use-cases/demos/classification/tensorflow-keras/entrypoint.sh:17]: this is ok, just FYI- i usually prefer to have default value set first and then just overwrite the value if USE_TFLITE == 1

**Review by NeethuES-intel — APPROVED** [review]
Everything worked. I see better fps on my machine with tflite than tensorflow.
